.github/workflows/pullrequest.yml

```diff
@@ -2,7 +2,9 @@ name: ci
 
 on:
   pull_request:
-
+  push:
```

Comment:
This will also trigger when the PR is merged in the main branch.
Is it what you want to do?

Reply:
Maybe the name `build.yml` would make more sense for both cases.
What do you think?

Reply:
sure, renamed: [`125b3e3` (#42)](https://github.com/bucketeer-io/javascript-client-sdk/pull/42/commits/125b3e3cea5526722cef0bb50931a6ca48f3fa8a)